make_index.py

```diff
@@ -117,9 +117,6 @@ def main(argv: Sequence[str] | None = None) -> int:
             )
         )
 
-    # for now we don't utilize the json api
-    shutil.rmtree(os.path.join(args.dest, "pypi"), ignore_errors=True)
-
```

Comment:
so there's another problem with this -- which is why I disabled them to start with -- is I couldn't figure out how to make gcs serve them with the proper content type

they're also like 95%+ of the output data